src/os/bluestore/BlueStore.cc

```diff
@@ -6999,7 +6999,7 @@ int BlueStore::_setup_block_symlink_or_file(
       // a transport id for PCIe looks like: "trtype:PCIe traddr:0000:02:00.0"
       // where "0000:02:00.0" is the selector of a PCI device, see
       // the first column of "lspci -mm -n -D"
-      // a transport id for tcp looks like: "trype:TCP adrfam:IPv4 traddr:172.31.89.152 trsvcid:4420"
+      // a transport id for tcp looks like: "trtype:TCP adrfam:IPv4 traddr:172.31.89.152 trsvcid:4420"
```

Comment:
?

Reply:
Given the preceding lines, I think this is valid.